python/yolov7.py

```diff
@@ -240,12 +243,25 @@ def infer_cam(self, source):
         self.infer_queue.set_callback(self.postprocess)
         # Capture camera source
         cap = cv2.VideoCapture(source)
+        fourcc = cv2.VideoWriter_fourcc(*'XVID')
+        fps = int(round(cap.get(cv2.CAP_PROP_FPS)))
+        width = int(cap.get(cv2.CAP_PROP_FRAME_WIDTH))
+        height = int(cap.get(cv2.CAP_PROP_FRAME_HEIGHT))
+        out = cv2.VideoWriter('resultsvideo.avi',fourcc,fps,(width,height))
```

Comment:
If there is a live streaming input, I think recording it as a video is unnecessary, or we should let user to make the decision, but not hard code on it.

Reply:
thanks, ill try to fix it & also im not getting output as detected result, what should i need to pass in out.write( )

when i pass img/frame its not saving detected video